draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3581,13 +3663,13 @@ Server: example-server/0.1
    </figure>
    </section>
 
-   <section title="Invalidate" anchor="examples-invalidate2"><t>
-   Below is an example of a CI/T "invalidate" trigger -- another POST to
-   the collection of all triggers. This instructs the
-   dCDN to revalidate the content at "https://www.example.com/a/index.html", as well as any metadata and content whose URLs are
-   prefixed by "https://metadata.example.com/a/b/" using
-   case-insensitive matching, and "https://www.example.com/a/b/" using
-   case-sensitive matching, respectively.</t>
+   <section title="Invalidate" anchor="examples-invalidate2">
+   <t>
+   Below is an example of a CI/T "invalidate" trigger creation. 
+   This trigger instructs the dCDN to revalidate the content at "https://www.example.com/a/index.html", 
+   as well as any metadata and content whose URLs are prefixed by "https://metadata.example.com/a/b/" using
```

Comment:
This is slightly incorrect.  It should describe metadata that is prefixed by the URL using case-insensitive match, and content that is prefixed by the URL using case-sensitive matching.

Reply:
rewrote using <list> for clarity